VimeoUpload/Upload/Descriptor System/Cameo/CAMUploadDescriptor.swift

```diff
@@ -0,0 +1,356 @@
+//
+//  CAMUploadDescriptor.swift
+//  Cameo
+//
+//  Created by Westendorf, Michael on 6/27/16.
+//  Copyright © 2016 Vimeo. All rights reserved.
+//
+
+import Foundation
+import AFNetworking
+import VimeoNetworking
+
+public class CAMUploadDescriptor: ProgressDescriptor, VideoDescriptor
+{
+    let videoUrl: NSURL
+    let videoSettings: VideoSettings?
+    let thumbnailFileUrl: NSURL?
+    
+    private(set) var uploadTicket: VIMUploadTicket?
+    private(set) var video: VIMVideo?
+    private(set) var thumbnailTicket: VIMThumbnailUploadTicket?
+    private(set) var thumbnail: VIMPicture?
```

Comment:
Nit: should we call the var `picture` and `pictureTicket`? I guess it's too much for right now